guizero/alerts.py

```diff
@@ -1,14 +1,58 @@
 from tkinter import messagebox
 # Generates alerts
 
+
 def warn(title, text):
-    messagebox.showwarning(title,text)
+    """
+    Alert type warning.
+
+    :param string title:
+        String with the title to be apply.
+    :param string text:
+        String with the text to be displayed.
+    :return:
+        Nothing.
+    """
+    messagebox.showwarning(title, text)
+
 
 def info(title, text):
-    messagebox.showinfo(title,text)
+    """
+    Alert type info
+
+    :param string title:
+        String with the title to be apply.
+    :param string text:
+        String with the text to be displayed.
+    :return:
+        Nothing.
+    """
+    messagebox.showinfo(title, text)
+
 
 def error(title, text):
+    """
+    Alert type error
+
+    :param string title:
+        String with the title to be apply.
+    :param string text:
+        String with the text to be displayed.
+    :return:
+        Nothing.
+    """
     messagebox.showerror(title, text)
 
+
 def yesno(title, text):
+    """
+    Alert type yes or no
```

Comment:
Perhaps:

"""
Display a yes / no message box.

Returns `True` if the "yes" selected, `False` if "no".
"""